Dockerfile

```diff
@@ -1,22 +1,16 @@
-# WARNING: We must pin the Alpine version to 3.8 because freeradius-rest (r6) package in alpine-3.9 contains
-# a regression breaking validation of rest authentication responses. Causes health checks to fail.
-# This is a known issue with FreeRadius: https://github.com/FreeRADIUS/freeradius-server/issues/2821
-FROM ruby:2.6.3-alpine3.8
+FROM govwifi-frontend-base:latest
```

Comment:
This is purely for my curiosity. Where and how do we get this image? I can see it's from Dockerfile.base. Do we need to update the documentation in the repo to reflect the new process?

Reply:
Never mind, I found the other PR: https://github.com/alphagov/govwifi-concourse-deploy-pipeline/pull/14/files